testing/TEST-XSPEC-SET.xpl

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<p:declare-step xmlns:p="http://www.w3.org/ns/xproc"
+   xmlns:c="http://www.w3.org/ns/xproc-step" version="3.0"
+   xmlns:ox="http://csrc.nist.gov/ns/oscal-xproc3"
+   type="ox:TEST-XSPEC-SET"
+   name="TEST-XSPEC-SET"
+   >
+
+   <!-- Pipeline to be called as subpipeline, delivering a list of XProcs -->
+   
+   <p:input port="source" sequence="true">
+      <!-- We need content type because xspec suffix throws off the parser -->
+      <p:document href="../smoketest/congratulations-xslt.xspec" content-type="application/xml"/>
```

Comment:
As an experiment, I changed href to point to `../../xspec/tutorial/escape-for-regex.xspec` in my local XSpec clone and then ran `xp3 testing\BATCH-XSPEC-JUNIT.xpl`. (Note that that particular .xspec test has one verification failure.) The test ran, but the JUnit file I got was `testing/xspec-reports/_junit.xml` with a degenerate file name. Maybe what I did isn't meant to work, but I wanted to let you know just in case it should have worked.

Is there supposed to be a way for me to use one of these pipelines to run one or more XSpec tests that I specify on the command line? Maybe that's either a future enhancement or out of scope for this project.